src/tiktokbot.ts

```diff
@@ -0,0 +1,283 @@
+// Import modules.
+import { config } from 'dotenv'
+import { Client, Message, Options, MessageAttachment, MessageEmbed } from 'discord.js'
+import { getVideoMeta as fetch } from 'tiktok-scraper'
+
+// Log <ENV> errors.
+if (config()?.error) throw new Error('Failed to load .env file')
+
+// Default embed color.
+const color: number | string = 0xE18499
+
+// Define global interfaces.
+declare global {
+    var Client: object
+
+    interface String {
+
+        /**
+         * Compare two strings case insensitive.
+         * 
+         * @param [query] The string to compare with "this".
+         * @return boolean
+         * 
+         * @example
+         * "a".is("b"): false (boolean)
+         * "a".is("A"): true (boolean)
+         */
+
+        is(query: string): boolean
+    }
+
+    interface MessageExtender extends Message {
+        args: string[]
+        command: string
+    }
+}
+
+/**
+ * Create and login to Discord client with custom properties.
+ * 
+ * @param [] None required.
+ * @example new TikTok()
+ */
+export default class TikTok extends Client {
+    public constructor() {
+        super({
+            intents: ['GUILDS', 'GUILD_MESSAGES'],
+            makeCache: Options.cacheWithLimits({ MessageManager: { maxSize: 200, sweepInterval: 5 * 60000 } }),
+            presence: { status: 'dnd', activities: [{ name: 'Just Started Up. What Did I Miss? | /help', type: 'LISTENING' }] }
```

Comment:
Everything else looks good to go, just wondering is there an external handling `/help` or does that need to be replaced? lmao

Reply:
[Oops, Forgot to commit lol](https://github.com/txj-xyz/TikTokBot/pull/2/commits/6ec3ec83f2b0c39a2e3fe604234b7dcdf15cc095)
![Status](https://i.imgur.com/bnwbKSo.png)